src/Ztm.Zcoin.Rpc/ZcoinRpcClient.cs

```diff
@@ -139,16 +132,17 @@ class BalanceDetail
                 throw new ArgumentNullException(nameof(property));
             }
 
-            var resp = await this.client.SendCommandAsync("exodus_getbalance", address.ToString(), property.Id.Value);
-            var balanceDetail = JsonConvert.DeserializeObject<BalanceDetail>(resp.ResultString);
+            var resp = await this.client.SendCommandAsync("exodus_getbalance", address.ToString(), ToNative(property.Id));
+            var rawBalance = resp.Result.Value<string>("balance");
+            var rawReserved = resp.Result.Value<string>("reserved");
 
             var balance = property.Type == PropertyType.Divisible
-                ? PropertyAmount.FromDivisible(Convert.ToDecimal(balanceDetail.Balance))
-                : new PropertyAmount(Convert.ToInt64(balanceDetail.Balance));
+                ? PropertyAmount.FromDivisible(Convert.ToDecimal(rawBalance))
+                : new PropertyAmount(Convert.ToInt64(rawBalance));
 
             var reserved = property.Type == PropertyType.Divisible
-                ? PropertyAmount.FromDivisible(Convert.ToDecimal(balanceDetail.Reserved))
-                : new PropertyAmount(Convert.ToInt64(balanceDetail.Reserved));
+                ? PropertyAmount.FromDivisible(Convert.ToDecimal(rawReserved))
+                : new PropertyAmount(Convert.ToInt64(rawReserved));
 
             return (balance, reserved);
```

Comment:
Better to use named field instead so it will not break when the order is changed (e.g. `(Balance: balance, Reserved: reserved)`).

Reply:
good idea